gradle-sls-packaging/build.gradle

```diff
@@ -38,6 +37,18 @@ pluginBundle {
             displayName = 'Creates SLS pod distributions'
         }
     }
+
+    // Fix for publishPlugins not working with dependency recommenders (nebula, in our case).
+    // Gradle suggests the 'withDependencies' approach in the following workaround:
+    // http://plugins.gradle.org/help/plugin/missing-dependency-version
+    // TODO(dsanduleac): this is bad and we want to remove it
```

Comment:
I don't really want to introduce hacks and TODOs unless there is a clear way that future maintainers can figure out whether the hack can be removed.  For example, if there's an open issue we could link to that (when resolved) would allow us to remove this, then it would feel more OK.